nexus/posts/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from . import models
+
+
+class PostSerializer(serializers.ModelSerializer):
+    posted_by = serializers.ReadOnlyField(source='posted_by.email')
```

Comment:
This can be done by updating the str representation of user to have email. 

If you don't want them, use `posted_by__email`. Don't use `.` operator.

Reply:
String representation would make more sense.

Reply:
Okay. So changing `str` for `Users` model wouldn't change their serialized representation. I had to use `posted_by.email`. Also, `posted_by__email` doesn't work.